src/Structures/MDClient.js

```diff
@@ -29,6 +28,10 @@ module.exports = class MenuDocsClient extends Client {
 		if (!options.prefix) throw new Error('You must pass a prefix for the client.');
 		if (typeof options.prefix !== 'string') throw new TypeError('Prefix should be a type of String.');
 		this.prefix = options.prefix;
+		
+		if (!options.owners) throw new Error('You must pass a list of owners for the client.');
+		if (typeof options.owners !== 'object') throw new TypeError('Owners should be a type of Array<String>.');

```

Comment:
You should use `Array.isArray(options.owners)`

Reply:
Oh, you're so right!!

Reply:
I can't make that commit as I'm in my bed and about to sleep, but if you wish to, please do